### ISSUE TITLE
Is it available for Chinese Characters?

### ISSUE THREAD

**Issue by wangay21** [Merck/r2rtf#89]
I saved Rscript with UTF-8 to allow Chinese Character then it will report as below
![image](https://user-images.githubusercontent.com/75113143/154184131-5be4337d-61f3-4594-8511-3dda165e6c10.png)
It seems that I cannot input footnotes in Chinese, so I removed these footnotes, then it could output an RTF file but with all Chinese strings are unreadable.

**Comment by elong0527** [comment]
Could you provide an example using `reprex`? https://reprex.tidyverse.org/

**Comment by elong0527** [comment]
Closing the issue. 

With a recent fix in #93, we can use code below for the table. 

```
library(r2rtf)
library(dplyr)
head(iris) %>% 
  rtf_title(r2rtf:::utf8Tortf("标题: abc")) %>% 
  rtf_body() %>% 
  rtf_footnote(r2rtf:::utf8Tortf("中文脚注: abc")) %>%
  rtf_encode() %>% 
  write_rtf("tmp.rtf")
```
